lib/collectionspace/converter/core/organization.rb

```diff
@@ -20,12 +20,12 @@ def convert
                 "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance"
             ) do
               xml.parent.namespace = nil
-              Contact.map(xml, attributes)
+              Contact.map_contact(xml, attributes)
```

Comment:
Because this is core, nothing will be overridden, so we can call this directly instead of with a self.map_contact method